lizard_downloader.py

```diff
@@ -216,7 +217,7 @@ def show_data(self):
         """ Show the data as a new layer on the map """
 
         # Setup
-        WGS84 = "EPSG:4326"
+        wgs84 = "EPSG:4326"
```

Comment:
why is this changed to lower_case? It's a constant so upper case is better I think.